File: content/how-to/market-hours/examples/marketHours.py
import alpaca_trade_api as tradeapi

api = tradeapi.REST()

# Check if the market is open now.
clock = api.get_clock()
print('The market is {}'.format('open.' if clock.is_open else 'closed.'))

# Check when the market was open on Dec. 1, 2018
date = '2018-12-01'
calendar = api.get_calendar(start=date, end=date)
print('The market opened at {} and closed at {} on {}.'.format(
    calendar.open,
    calendar.close,
    date
))

File: content/how-to/market-hours/examples/marketHours.js
const Alpaca = require('@alpacahq/alpaca-trade-api')
const alpaca = new Alpaca()

// Check if the market is open now.
alpaca.getClock().then((clock) => {
    console.log('The market is ' + clock.is_open ? 'open.' : 'closed.')
})

// Check when the market was open on Dec. 1, 2018.
const date = '2018-12-01'
alpaca.getCalendar({
    start: date,
    end: date
}).then((calendar) => {
    console.log(`The market opened at ${calendar.open} and closed at ${calendar.close} on ${date}.`)
})